schemas/com.snowplowanalytics.mobile/application/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an application context",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"version": {
+			"type": "string",
+			"description": "Version of the application e.g 1.1.0",
+			"maxLength": 64
```

Comment:
I think 16 or 32 should be enough for longest types of versioning.

Reply:
will amend